pod/perl5121delta.pod

```diff
@@ -10,7 +10,7 @@ This document describes differences between the 5.12.0 release and
 the 5.12.1 release.
 
 If you are upgrading from an earlier release such as 5.10.1, first read
-L<perl5120delta>, which describes differences between 5.10.1 and
+L<perl5120delta>, which describes differences between 5.10.0 and
```

Comment:
I don't think this is correct. Versioning worked differently from how it does not back then.

Reply:
perl5120delta does seem to include the changes included in 5.10.1, so this change seems fine to me.

Reply:
perl5120delta.pod itself says:

> This document describes differences between the 5.10.0 release and the
> 5.12.0 release.

so I went by that. But it also says:

> Many of the bug fixes in 5.12.0 are already included in the 5.10.1
> maintenance release.

Does perl5120delta.pod need to be updated (also or instead)? I haven't looked into the details.